joplin/templates/joplin_UI/primary_navigation.html

```diff
@@ -0,0 +1,31 @@
+{% load staticfiles wagtailadmin_tags %}
+{% load i18n %}
+
+{% block content %}
+
+<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+<link href="https://fonts.googleapis.com/css?family=Medium+IBM+Plex+Sans&display=swap" rel="stylesheet">
+
+<aside id="coa-primary_navigation" class="nav-wrapper" style="display: none;" data-nav-primary>
```

Comment:
I'm curious, is this stuff kinda copied over from the wagtail template? If all were doing is adding the ID, is there a way we could do that by selecting this element with javascript and then adding the ID? Whatever we can do to avoid having to maintain extra templates would probably be good..or at least note which template in wagtail this is copied from. :-D

Reply:
So, my thinking here was driven by a few reasons. 

Because we want to change the style and behavior of a specific template, it's nice to have any reference to anything we've modified within our code base. 

But, beyond that, it should prevent updates from changing our html code. This happened to a few things when we updated wagtail last time, where the html of a template  changed it structure and broke our targeted css modifications. 

Without having it in our control, our entire UI is suspect to break with any minor change to the wagtail library. So, If we change it. we must own it! 

Also, in the future when somebody sees where we modified some css, they can find it within our code base easily. 

-
It's a good question, and if you think that makes since, i'll add this explanation to the documentation.